### PR TITLE
Allows empty attributes in html tag helper

### PR DESCRIPTION
**Pull request by Xaviju**
Fixes this issue -> https://github.com/hexojs/hexo/issues/3222

### CODE REVIEW

File: lib/html_tag.js [hunk 1]
```diff
@@ -6,7 +6,7 @@ function htmlTag(tag, attrs, text) {
   var result = '<' + tag;
 
   for (var i in attrs) {
-    if (attrs[i]) result += ' ' + i + '="' + attrs[i] + '"';
+    if (attrs[i] || attrs[i] === '') result += ' ' + i + '="' + attrs[i] + '"';
```
Comment (JLHwung) [inline lib/html_tag.js]: I suggest we use stricter `attrs[i] !== undefined && attrs[i] !== null` on this check.

Comment (Xaviju) [reply lib/html_tag.js]: There is an error already if an attribute is not defined.  Should I add the test?
```
it('tag + empty attr', function() {
    try {
      htmlTag('img', {
        src: 'http://placekitten.com/200/300',
        hidden
      });
    } catch (err) {
      err.should.have.property('message', 'hidden is not defined');
    }
  });
```
or
```
it('tag + empty attr', function() {
    try {
      htmlTag('img', {
        src: 'http://placekitten.com/200/300',
        alt: undefined
      });
    } catch (err) {
      err.should.have.property('message', 'alt is not defined');
    }
  });
```

**Comment by Xaviju** [comment]
Thinking it twice, just adding the conditions you suggested, all possible valid attributes are working as expected. I've added a few tests to ensure that works properly.

**Comment by NoahDragon** [comment]
Could you please fix the error in travis-ci?

**Comment by Xaviju** [comment]
@NoahDragon should be fixed now

**Comment by NoahDragon** [comment]
@Xaviju Great, thank you.

**Pull request merged**